prepare-for-current-kernel.sh

```diff
@@ -7,14 +7,14 @@ then
 elif { echo $VERSION ; echo "5.99" ; } | sort -V -c 2>/dev/null
 then
   PATCHFILE="patch5.8"
-elif { echo $VERSION ; echo "6.19" ; } | sort -V -c 2>/dev/null
+elif { echo $VERSION ; echo "6.10" ; } | sort -V -c 2>/dev/null
 then
   PATCHFILE="patch6.0"
 else
-  PATCHFILE="patch6.2"
+  PATCHFILE="patch6.11"
```

Comment:
Is there a reason you removed the branch for 6.2?

Reply:
Hi Plippo no reason it was a quick way to test and validate the patch when I tested it on my Zenbook. I actually intended to only submit the patch file and not `prepare-for-current-kernel.sh`. I'll remove it and you can adjust the `prepare-for-current-kernel.sh` accordingly.

Reply:
Thank you for the clarification! No need to remove it, I will adjust it directly.